examples/supermariobox/populate_mario_box.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+import base64
+import json
+import os
+
+import click
+
+from nucypher.cli.config import nucypher_click_config
+from nucypher.characters.lawful import Enrico
+from nucypher.cli.actions import make_cli_character
+from nucypher.config.characters import AliceConfiguration
+from nucypher.crypto.powers import SigningPower
+
+
+@click.command()
+@click.option('--plaintext-dir', type=click.STRING, required=True)
+@click.option('--outfile', type=click.STRING)
+@click.option('--alice-config', type=click.STRING)
+@click.option('--label', type=click.STRING, required=True)
+@nucypher_click_config
+def mario_box_cli(click_config, plaintext_dir, alice_config, label, outfile):
+
+    # Derive Policy Encrypting Key
+    alice_configuration = AliceConfiguration.from_configuration_file(filepath=alice_config)
+    alice = make_cli_character(character_config=alice_configuration, click_config=click_config)
+    alice_signing_key = alice.public_keys(SigningPower)
+    policy_encrypting_key = alice.get_policy_encrypting_key_from_label(label=label.encode())
+    policy_encrypting_key_hex = bytes(policy_encrypting_key).hex()
```

Comment:
Since the latest release of umbral you can directly do:
```suggestion
    policy_encrypting_key_hex = policy_encrypting_key.hex()
```